cryosparc2/protocols/protocol_cryosparc2d.py

```diff
@@ -76,7 +77,7 @@ def _defineParams(self, form):
         # ----------- [2D Classification] --------------------------------
 
         form.addSection(label="2D Classification")
-        form.addParam('numberOfClasses', IntParam, default=50,
+        form.addParam('class2D_K', IntParam, default=50,
```

Comment:
just be aware that changing the names will affect users with old projects

Reply:
Yes, in principle it would not produce errors when loading the protocols, it would just load the default values.... and that is an error....... I will change the parameter names again. Thanks, @azazellochg